spec/requests/api/v0/university_profile_retrieves_keys_spec.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.describe 'University Profile retrieves research keys', type: :request do
```

Comment:
change to "registration keys" for consistency

Reply:
The naming convention for test descriptions is research keys